src/server/game/Accounts/RBAC.h

```diff
@@ -759,6 +759,8 @@ enum RBACPermissions
     RBAC_PERM_COMMAND_GO_OFFSET                              = 852,
 
     // custom permissions 1000+
+	RBAC_PERM_COMMAND_MODIFY_FURY                            = 1001,
```

Comment:
![correct](https://cloud.githubusercontent.com/assets/21317689/23871533/16039d54-0840-11e7-9c44-e5da1aebde55.PNG)
no problem in Visual studio, i dont know how to fix this tab Oo